modules/ucsm.py

```diff
@@ -0,0 +1,34 @@
+from ucsmsdk.ucshandle import UcsHandle
+from ucsmsdk.ucsexception import UcsException
+import master_password
+
+class UcsmServer:
+    def __init__(self, ucsm_server, user, master_password):
+        self.ucsm_server = ucsm_server
+        self.user = user
+        self.master_password = master_password
+        self.password = self._get_password()
+        self.handle = self._login()
+
+    def _get_password(self):
```

Comment:
this could be something generic which is going to be needed elsewhere too. Either put it in a helper class or in the BaseClass, if you decide to go for it. I then would give `UcsmServer` the decrypted pw straight.